code/script.js

```diff
@@ -1,9 +1,15 @@
-// All the DOM selectors stored as short variables
+// DOM selectors 
```

Comment:
In general you had great structure for your comments on each sections! This made it easy to follow the code and know what the different sections were for! 👏